src/kem/sike/P503/P503.c

```diff
@@ -108,15 +108,15 @@ static const unsigned int strat_Bob[MAX_Bob - 1] = {
 #define mp_add_asm mp_add503_asm
 #define mp_addx2_asm mp_add503x2_asm
 #define mp_subx2_asm mp_sub503x2_asm
-#define crypto_kem_keypair oqs_kem_sike_p503_keypair
-#define crypto_kem_enc oqs_kem_sike_p503_enc
-#define crypto_kem_dec oqs_kem_sike_p503_dec
-#define random_mod_order_A oqs_kem_sike_p503_sidh_random_mod_order_A
-#define random_mod_order_B oqs_kem_sike_p503_sidh_random_mod_order_B
-#define EphemeralKeyGeneration_A oqs_kem_sike_p503_sidh_EphemeralKeyGeneration_A
-#define EphemeralKeyGeneration_B oqs_kem_sike_p503_sidh_EphemeralKeyGeneration_B
-#define EphemeralSecretAgreement_A oqs_kem_sike_p503_sidh_EphemeralSecretAgreement_A
-#define EphemeralSecretAgreement_B oqs_kem_sike_p503_sidh_EphemeralSecretAgreement_B
+#define crypto_kem_keypair OQS_KEM_sike_p503_keypair
+#define crypto_kem_enc OQS_KEM_sike_p503_enc
+#define crypto_kem_dec OQS_KEM_sike_p503_dec
+#define random_mod_order_A OQS_KEM_sike_p503_sidh_random_mod_order_A

```

Comment:
Would you have a chance to make one small change -- keep the keypair/enc/dec ones as you've got them (`OQS_KEM_sike_p503_keypair`), but do the rest (random_mod_order_A, etc.) in lowercase (`oqs_kem_sike_p503_sidh_random_mod_order_A`)?  The idea is that "public functions in the API" start with capital letters, but non-public API functions (still properly namespaced) are lowercase.